ajax/libs/jquery.complexify.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery.complexify.js",
+  "filename": "jquery.complexify.min.js",
+  "version": "0.5.1",
+  "description": "JavaScript library for inform users of password strength",
+  "homepage": "http://danpalmer.me/jquery-complexify",
```

Comment:
Sorry, I misread it last time. It is okay now.
@PeterDaveHello Please help me check other part. Thank you!